ani-cli

```diff
@@ -216,7 +216,12 @@ update_history() {
 download() {
     case $1 in
         *m3u8*) ffmpeg -loglevel error -stats -i "$1" -c copy "$download_dir/$2.mp4" ;;
-        *) aria2c --check-certificate=false --continue --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide ;;
+        *) if uname -a| grep -q "ish"
+            then
+            curl --output-dir "$download_dir" -o "$2.mp4" "$1"
+            else  
+            aria2c --check-certificate=false --continue --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide 
+            fi ;;
```

Comment:
This warrants a discussion about our use of aria2.
We may choose to use curl if no significant speed gains can be seen otherwise